doc/whats_new/v0.24.rst

```diff
@@ -256,6 +256,18 @@ Changelog
   :pr:`18280` by :user:`Alex Liang <tianchuliang>` and
   `Guillaume Lemaitre`_.
 
+- |Enhancement| :func:`datasets.fetch_openml` now supports categories with
+  missing values when returning a pandas dataframe. :pr:`18383` by
+  `Thomas Fan`_.
+
+- |Feature| :func:`datasets.fetch_openml` now validates md5checksum of arff
```

Comment:
This entry was already part of the previous release. You can remove it.

Reply:
done